.pre-commit-config.yaml

```diff
@@ -1,24 +1,46 @@
 repos:
--   repo: https://github.com/psf/black
+  - repo: https://github.com/psf/black
     rev: 22.3.0
     hooks:
-    - id: black
-      args: ["cfgov", "--line-length=79"]
-      exclude: migrations
--   repo: https://gitlab.com/pycqa/flake8
+      - id: black
+        args: ['cfgov', '--line-length=79']
+        exclude: migrations
+  - repo: https://gitlab.com/pycqa/flake8
```

Comment:
Can we update this to `https://github.com/PyCQA/flake8`?

cc @willbarton for awareness. Does this need to be pulled from gitlab for some reason?